.erda/migrations/cmdb/20211214-issue-state-circulation.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE `dice_issue_state_circulation`
```

Comment:
erda_issue_state_circulation

Reply:
The others are prefixed with dice like issue,issue_state and so on

Reply:
done